### ISSUE TITLE
bug(dzongkha): Dzongkha keyboard shows double dotted circle on shift layer on Android

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyboards#1754]
One thing that stays the same is that the shift layer of Dzongkha keyboard is not shown correctly. No double dotted circle should be shown.

<img width="992" alt="image" src="https://user-images.githubusercontent.com/28331388/144842296-84d31696-7e92-4d83-8347-ce97a3497b3b.png">

_Originally posted by @MakaraSok in https://github.com/keymanapp/keyman/issues/5906#issuecomment-986709004_

Not clear to me if this is limited to KeymanWeb on Android Chrome or if it arises elsewhere also?

**Comment by mcdurdin** [comment]
This looks like a font issue. Consistency between platforms might require a custom font though 😢

**Comment by DavidLRowe** [comment]
@mcdurdin Is this related to #2172 ?

**Comment by mcdurdin** [comment]
Yes, this relates to #2172 but needs to be fixed as a separate task, so we should leave it open. We can't address until we are using the v17 compiler toolchain so it'll be a little while away.

**Comment by LornaSIL** [comment]
@mcdurdin  I'm not seeing the double 25cc anymore. Either this issue was fixed in Keyman 17, or Android fixed it? I'm on Android 14.
Can we close the issue?
![Image](https://github.com/user-attachments/assets/15e73a18-fb3d-4c2e-928a-33e83fedee45)

**Comment by mcdurdin** [comment]
> [@mcdurdin](https://github.com/mcdurdin) I'm not seeing the double 25cc anymore. Either this issue was fixed in Keyman 17, or Android fixed it? I'm on Android 14.
> Can we close the issue?

Yes looks to be resolved on my Android 14 phone also. Closing.
